tests/unit/plugins/test_writer.py

```diff
@@ -244,10 +244,21 @@ def check_requirements_txt(t1: str, t2: str):
             "RAY",
             "ray_pipeline_a0_b0_task_session",
             {},
-            {"dag_flavor": "TaskPerSession"},
+            {"dag_flavor": "TaskPerSession", "use_workflows": False},
```

Comment:
I'm surprised by this change. can you point to the documentation that says that default it True? Its an alpha version so not sure why its the default.

Reply:
I think its because its more strategic for us to use Ray Workflows so I chose to make Lineapy's default use workflows if not specified.  

@moustafa-a can be the final decision maker here.

Reply:
clarified in code that this is a linea configuration, not passed to ray.init

Reply:
resolved using nice words.